src/modules/exec.ts

```diff
@@ -151,7 +152,7 @@ async function createPythonVenv(
     cwd: string,
     loggingOptions: { writeStdout?: boolean; verbose?: boolean },
 ) {
-    const venvDir = `${ProjectCache.getCacheDir()}/pyvenv/${componentId}`;
+    const venvDir = `${getPackageFolderPath()}/.pyvenvs/${componentId}`;
```

Comment:
Fixes just the Python venv issue.

Reply:
I would argue this is not a package and should not go in the package path.

My suggestion would be to move all project related data which is right now put into `$VELOCITAS_HOME/projects/<HASH>` directly into the project's workspace under a hidden folder `.velocitas` that way the user is able to freely rename or move his project and still keep all of his cache and project data intact.

This would even be an easy fix by introducing and environment variable called `VELOCITAS_INLINE_PROJECT` which, if set, redirects the output which would usually go to `$VELOCITAS_HOME/projects/<HASH>` to `$VELOCITAS_WORKSPACE/.velocitas`